npx-jest-results.txt

```diff
@@ -0,0 +1,151 @@
+❯  npx jest
+ FAIL  out/test/jest/extension.test.js
+  ✓ foo (2 ms)
+  ✓ slugifyMethod (1 ms)
+  ✕ noteFileNameFromTitle (3 ms)
+  ✓ rxTagNoAnchors (1 ms)
+  ✓ rxWikiLink (1 ms)
+  ✕ noteNamesFuzzyMatch (1 ms)
+  ✕ noteNamesFuzzyMatchSlashes (1 ms)
+  ✓ noteNamesFuzzyMatch (1 ms)
+  ✓ titleCaseFromFilename
+  ✕ Note._rawRangesForWord (4 ms)
+  ✓ Note.tagSet (1 ms)
+  WikiLinks
+    ✓ cleanPipedWikiLinks (1 ms)
+    ✕ NoteWorkspace.noteNamesFuzzyMatch
+    ✕ Config changes (1 ms)
+  NoteWorkspace.newNoteContent
+    ✓ handles noteName tag
+    ✓ handles escaped newlines (1 ms)
+    ✓ handles timestamp
+    ✓ handles date
+
+  ● noteFileNameFromTitle
+
+    expect(received).toEqual(expected) // deep equality
+
+    Expected: "some title.md"
+    Received: "some-title.md"
+
+      29 |   let orig = NoteWorkspace.slugifyChar;
+      30 |   NoteWorkspace.slugifyChar = (): string => 'NONE';
+    > 31 |   expect(NoteWorkspace.noteFileNameFromTitle('Some Title')).toEqual('some title.md');
```

Comment:
This test breaks because `github-slugger` does not support the `slugifyChar`  = `NONE` 

Should we fix by:
1. Warning you that `github-slugger` is only compatible with `slugifyChar`  = `-`
2. Replacing `-` with ` ` after calling `github-slugger`

I lean towards (1)